models/DataObject/ClassDefinition.php

```diff
@@ -383,8 +383,7 @@ public function save(bool $saveDefinitionFile = true): void
 
         foreach ($fieldDefinitions as $fd) {
             // call the method "classSaved" if exists, this is used to create additional data tables or whatever which depends on the field definition, for example for localizedfields
-            //TODO Pimcore 11 remove method_exists call
-            if (!$fd instanceof ClassDefinition\Data\DataContainerAwareInterface && method_exists($fd, 'classSaved')) {
+            if ($fd instanceof \Pimcore\Model\DataObject\ClassDefinition\Data\ClassSavedInterface) {
```

Comment:
Hi @robertSt7, please have a look again, for example Localized fields are not implementing ClassSavedInterface
https://github.com/pimcore/pimcore/blob/11.x/models/DataObject/ClassDefinition/Data/Localizedfields.php#L31